front/components/navigation/Navigation.tsx

```diff
@@ -24,10 +28,10 @@ export function Navigation({
   subNavigation,
 }: NavigationProps) {
   const { sidebarOpen, setSidebarOpen } = useContext(SidebarContext);
-
   if (hideSidebar) {
     return null;
   }
+  const [navigationBar, setNavigationBarOpen] = useState(true);
```

Comment:
We use `isXX` naming convention for boolean variables.
```suggestion
  const [isNavigationBarOpen, setNavigationBarOpen] = useState(true);
```

Reply:
Put this hook above the if block. Hook consistency if important between renderings, no matter the props value.